Competition/src/main/java/competition/subsystems/elevator/commands/ControlElevatorWithGamepadCommand.java

```diff
@@ -0,0 +1,27 @@
+package competition.subsystems.elevator.commands;
+
+import competition.operator_interface.OperatorInterface;
+import competition.subsystems.elevator.ElevatorSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class ControlElevatorWithGamepadCommand extends BaseCommand {
+    final OperatorInterface oi;
+    final ElevatorSubsystem elevatorSubsystem;
+
+    public ControlElevatorWithGamepadCommand(OperatorInterface oi, ElevatorSubsystem elevatorSubsystem) {
```

Comment:
You need @Inject before this line. You pretty much need it for all constructors.